tidy3d/components/simulation.py

```diff
@@ -301,13 +301,13 @@ def warn(istruct, side):
 
                 zipped = zip(["x", "y", "z"], sim_bound_min, struct_bound_min, val)
                 for axis, sim_val, struct_val, pml in zipped:
-                    if pml.num_layers > 0 and struct_val > sim_val:
+                    if pml.num_layers > 0 and struct_val > sim_val and pml != Absorber():
```

Comment:
if absorber.num_layers is different from default, this will not catch it.  better to use `isinstance(pm, Absorber)` it think

Reply:
good point